cabal-install/cabal-install.cabal

```diff
@@ -239,6 +245,11 @@ executable cabal
         zlib       >= 0.5.3    && < 0.7,
         hackage-security >= 0.5 && < 0.6
 
+    if flag(old-bytestring)
+      build-depends: bytestring <  0.10.2, bytestring-builder
+    else
+      build-depends: bytestring >= 0.10.2
```

Comment:
Apparently there is already a `bytestring` dependency in the main `build-depends` list: `bytestring >= 0.9 && < 1`. The dependency on `bytestring-builder` should be `>= 0.10 && < 1`, IMO.